docs/agent/config/v3.mdx

```diff
@@ -344,17 +344,17 @@ ngrok start --all
-| [`upstream.protocol`](#upstreamprotocol)             |          | HTTP, HTTPS |                                                                                                                                                                        |
-| [`upstream.proxy_protocol`](#upstreamproxy_protocol) |          | HTTP, HTTPS | The version of [PROXY protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) to use with this endpoint, empty if not using. Example values are 1 or 2. |
+| Name                                                 | Required | Description                                                                                                                                                                |
+| ---------------------------------------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`name`](#name)                                      | Required | A unique name for this endpoint's configuration.                                                                                                                           |
+| [`url`](#url)                                        |          | The address you would like to use to forward traffic to your upstream service. Leave empty to get a randomly assigned address.                                             |
+| [`metadata`](#metadata-1)                            |          | An arbitrary string of user-defined metadata for this endpoint.                                                                                                            |
+| [`description`](#description)                        |          | An arbitrary user-defined description about this endpoint.                                                                                                                 |
+| [`traffic_policy`](#traffic-policy)                  |          | Accepts a Traffic Policy configuration in YAML format.                                                                                                                     |
+| [`upstream`](#upstream)                              | Required | Upstream service configuration options.                                                                                                                                    |
+| [`upstream.url`](#upstreamurl)                       | Required | The address you would like to forward traffic to.                                                                                                                          |
+| [`upstream.protocol`](#upstreamprotocol)             |          | The application layer protocol the agent should use when talking to your upstream application for HTTP endpoints.                                                          |
```

Comment:
Suggest adding something like "Accepted values are `http1` (default) and `http2`" here.